src/app/components/core/scheduler/list-schedules/list-schedules.component.ts

```diff
@@ -60,6 +64,73 @@ export class ListSchedulesComponent implements OnInit {
     };
   }
 
+  public filterSouthAndNorth (schedules): void {
+    //  TODO: remove log
+    console.log('Schedules', schedules);
+    this.configService.getCategoryWithChildren().
+      subscribe(
+        (data) => {
+          /** request completed */
+          this.ngProgress.done();
+
+          const allCats  = [];
+          data['categories'].forEach(element => {
+            allCats.push({ key: element.key, children: element.children });
+          });
+
+          const sn = [];
+          const south = allCats.filter(el => el.key.toUpperCase() === 'SOUTH');
+          south.forEach(s => {
+            s.children.forEach( el => {
+              sn.push(el.key);
+            });
+          });
+          const north = allCats.filter(el => el.key.toUpperCase() === 'NORTH');
+          north.forEach(n => {
+            n.children.forEach( el => {
+              sn.push(el.key);
+            });
+          });
```

Comment:
I would love to move this to another private `function filterCategories(categories)` where `categories = data['categories']`

Reply:
Done